api/python/provisioner/commands/__init__.py

```diff
@@ -1163,6 +1163,30 @@ def _passwordless_ssh():
             _generate_ssh_config()
             _copy_minion_nodes()
 
+        # Update PATH just as a precaution
+        StateFunExecuter.execute(
+            'file.blockreplace',
+            fun_kwargs=dict(
+                name='~/.bashrc',
```

Comment:
It will work not for only shells since it might not be read by shell in some cases. @divya-vijay please create a task to address that better later. For now extended list of sudo commands should anyway fix the issue.